### DISCUSSION TITLE
Move workspace to output1 output2

### DISCUSSION THREAD

**Discussion by gruni1992** [i3/i3#5339] [Q&A] [answered]
According to the i3 documentation, I can specify multiple outputs in the move command to cycle through them or have one as a fallback as non-existent outputs are being skipped. However if I run the command `i3-msg move workspace to output eDP-1 DP-2-3` I get the output `[{"success":false,"error":"Could not get output from string \"eDP-1 DP-2-3\""}]`. If I just specify one of the outputs, the command works fine. What am I missing here?

**Comment by Airblader** [comment, marked as answer]
My guess is that you are on an older version of i3. I believe this feature was introduced with 4.20.
